db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
```

Comment:
This is name of service which it refers through Fog and OpenStack - https://github.com/fog/fog-openstack/blob/master/lib/fog/openstack/models/compute/service.rb#L9 . I use it to pair SystemService with CloudService.